package.json

```diff
@@ -39,6 +39,7 @@
     "db:bootstrap:admin": "node ./build/server/tools/bootstrapAdminCLI.js",
     "db:validation": "node ./build/server/tools/dataValidationCLI.js",
     "db:migrate": "node_modules/.bin/sequelize db:migrate",
+    "db:migrate:create": "npx sequelize-cli migration:generate --name",
     "db:migrate:ci": "cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub node_modules/.bin/sequelize db:migrate",
```

Comment:
renaming the migrate task I made awhile ago to match the syntax of the other ones